WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomAmounts/OrderCustomAmountsSection.swift

```diff
@@ -62,7 +62,7 @@ private extension OrderCustomAmountsSection {
         static let addCustomAmount = NSLocalizedString("Add Custom Amount",
                                                    comment: "Title text of the button that allows to add a custom amount when creating or editing an order")
         static let customAmounts = NSLocalizedString("orderForm.customAmounts",
-                                                     value: "Custom Amounts",
+                                                     value: "Custom amounts",
```

Comment:
This should be title case on iOS: pecCkj-eD-p2

```suggestion
                                                     value: "Custom Amounts",
```

It's also nicer for it to be consistent with the row above
![order-details-custom-amounts-capitalisation](https://github.com/woocommerce/woocommerce-ios/assets/2472348/369abb62-9c3f-4822-a438-64f1d67f6a66)